ui/app/src/components/Dialogs/ConfirmDialog.tsx

```diff
@@ -24,18 +23,64 @@ import { defineMessages, useIntl } from 'react-intl';
 import StandardAction from '../../models/StandardAction';
 import Dialog from '@material-ui/core/Dialog';
 import { useUnmount } from '../../utils/hooks';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import confirmGraphicUrl from '../../assets/confirm.svg';
 
 const messages = defineMessages({
-  ok: {
-    id: 'words.ok',
-    defaultMessage: 'Ok'
+  accept: {
+    id: 'words.accept',
+    defaultMessage: 'Accept'
   },
   cancel: {
     id: 'words.cancel',
     defaultMessage: 'Cancel'
   }
 });
 
+const confirmDialogStyles = makeStyles(() =>
+  createStyles({
+    dialog: {
+      '& .MuiPaper-root': {
+        maxWidth: '350px',
+        borderRadius: '20px'
+      }
+    },
+    dialogBody: {
+      backgroundColor: '#fff',
+      textAlign: 'center',
+      padding: '40px 20px 0 !important'
+    },
+    dialogTitle: {
+      fontSize: '16px',
+      fontWeight: 400,
+      lineHeight: '24px',
+      paddingTop: '35px',
+      paddingBottom: '5px',
+      letterSpacing: '0.15px'
```

Comment:
Use typography variant set to body1. From this, if anything, only the bottom margin should remain.